Wikipedia/UI-V5/UIView+WMFShadow.m

```diff
@@ -11,16 +11,11 @@
 @implementation UIView (WMFShadow)
 
 - (void)wmf_setupShadow {
-    self.clipsToBounds       = NO;
+    self.layer.masksToBounds = NO;
```

Comment:
is there a functional difference between setting the layer property and the `UIView` property? IIRC the `UIView` property is a proxy for `masksToBounds`, which is the case for many `UIView` properties.